homeassistant/components/wallbox/lock.py

```diff
@@ -36,6 +37,8 @@ async def async_setup_entry(
         )
     except InvalidAuth:
         return
+    except ConnectionError as exc:
```

Comment:
you're no longer raising connection error.

Reply:
Connection error is raised in the async_lock_unlock method in the coordinator. I'm raising PlatformNotReady here during setup to prevent hangs.

Reply:
`ConnectionError` needs to be caught in the coordinator update method also.

Reply:
@MartinHjelmare does this refer to the additional catch in __init__?

Reply:
Yes.